### PR TITLE
Adding index and sketch analyzer plugins for Timesketch.

### PR DESCRIPTION
**Pull request by kiddinn**
Adding the ability to generate index and sketch analyzer plugins for Timesketch.

### CODE REVIEW

File: plasoscaffolder/templates/ts_index_analyzer.jinja2 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        super({{ class_name }}Plugin, self).__init__(index_name)
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Returns:
+            String with summary of the analyzer result
+        """
+        # TODO: Add Elasticsearch query to get the events you need.
+        # Documentation: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html
```
[the remark this answers is not in the document]
Comment (berggren) [reply plasoscaffolder/templates/ts_index_analyzer.jinja2]: Remove the documentation here for now. I will collect and point to internal docs later.

File: plasoscaffolder/templates/ts_sketch_analyzer.jinja2 [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        super({{ class_name }}SketchPlugin, self).__init__(sketch_id, index_name)
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Returns:
+            String with summary of the analyzer result
+        """
+        # TODO: Add Elasticsearch query to get the events you need.
+        # Documentation: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html
```
Comment (berggren) [inline plasoscaffolder/templates/ts_sketch_analyzer.jinja2]:
```suggestion
```

**Comment by kiddinn** [comment]
ignore failed travis, this is a known issue with an old test file, that is being removed in #40

File: plasoscaffolder/templates/ts_sketch_analyzer.jinja2 [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class {{ class_name }}SketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for {{ class_name }}."""
+
+    NAME = '{{ plugin_name }}'
+
+    def __init__(self, sketch_id, index_name):
```
Comment (berggren) [inline plasoscaffolder/templates/ts_sketch_analyzer.jinja2]:
```suggestion
    def __init__(self, index_name, sketch_id):
```

File: plasoscaffolder/templates/ts_sketch_analyzer.jinja2 [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        """Initialize The Sketch Analyzer.
+
+        Args:
+            sketch_id: Sketch ID
```
Comment (berggren) [inline plasoscaffolder/templates/ts_sketch_analyzer.jinja2]:
```suggestion
            index_name: Elasticsearch index name
```

File: plasoscaffolder/templates/ts_sketch_analyzer.jinja2 [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            index_name: Elasticsearch index name
```
Comment (berggren) [inline plasoscaffolder/templates/ts_sketch_analyzer.jinja2]:
```suggestion
            sketch_id: Sketch ID
```

**Pull request merged**